Signal/src/UserInterface/Strings.swift

```diff
@@ -15,16 +15,13 @@ import Foundation
 
     // MARK: Notification actions
     static let callBackButtonTitle = NSLocalizedString("CALLBACK_BUTTON_TITLE", comment: "notification action")
-    static let confirmIdentityAndCallBackButtonTitle = NSLocalizedString("CONFIRM_IDENTITY_AND_CALLBACK_BUTTON_TITLE", comment: "notification action, confirming that it's OK to proceed calling after a caller's Safety Number has changed")
     static let showThreadButtonTitle = NSLocalizedString("SHOW_THREAD_BUTTON_TITLE", comment: "notification action")
 
     // MARK: Missed Call Notification
-    static let missedCallNotificationBody = NSLocalizedString("MISSED_CALL", comment: "notification title")
-    static let missedCallNotificationBodyWithCallerName = NSLocalizedString("MSGVIEW_MISSED_CALL_WITH_NAME", comment: "notification title. Embeds {{Caller's Name}}")
```

Comment:
Got rid of a little duplication here. I didn't think it was worth distinguishing between copy which used *no* contact name, and that which used the "Signal User" generic name.